com.hoptodesk.HopToDesk.yml

```diff
@@ -0,0 +1,52 @@
+id: com.hoptodesk.HopToDesk
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: hoptodesk
+icon: share/icons/hicolor/scalable/apps/hoptodesk.svg
+appdata:
+  license: FSFAP
+  url: https://hoptodesk.com
+  metadata_license: FSFAP
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - xdotool.json
+  - name: HopToDesk
+    buildsystem: simple
+    build-commands:
+      - bsdtar -zxvf hoptodeskdeb.deb
+      - tar -xvf ./data.tar.xz
+      - cp -r ./usr/*  /app/
+      - ls /app/lib
+      - cp /app/lib/libsciter-gtk.so  /app/bin/
+      - chmod 755 /app/bin/libsciter-gtk.so
+      - mkdir -p /app/bin
+      - mv /app/share/applications/hoptodesk.desktop /app/share/applications/com.hoptodesk.HopToDesk.desktop
+      - sed -i '/^Icon=/ c\Icon=com.hoptodesk.HopToDesk' /app/share/applications/com.hoptodesk.HopToDesk.desktop
```

Comment:
modification to the desktop file should always be done with the `desktop-file-edit` command. See below for the note about using `rename-icon` as well.